.travis.yml

```diff
@@ -4,6 +4,14 @@ rust:
 branches:
   only:
     - bench
+before_install:
+  - curl -L https://downloads.sourceforge.net/project/gnuplot/gnuplot/5.0.6/gnuplot-5.0.6.tar.gz > gnuplot-5.0.6.tar.gz
+  - tar xzf gnuplot-5.0.6.tar.gz
+  - cd gnuplot-5.0.6
+  - ./configure
+  - make && make install
```

Comment:
Better:

Let Travis handle the installation for us.
https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-on-Container-Based-Infrastructure